### PR TITLE
Alec/tfac 749

### PR DESCRIPTION
**Pull request by amaliwanag2**

### CODE REVIEW

**Comment by amaliwanag2** [comment]
this is live on dev, along with the FE if you switch your cause ID or create a new user on dev and manually set cause id in local storage

File: graphql/database/cause/causes/blackEquity/causeData.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import onboardingTitle2 from './onboarding.step2.title.md'
+import onboardingTitle3 from './onboarding.step3.title.md'
+import socialTitle from './social.title.md'
+import socialSubtitle from './social.subtitle.md'
+import emailAbout from './social.email.about.md'
+
+const data = {
+  id: 'q1FFsAhbV',
+  about,
+  name: 'blackEquity',
```
Comment (kmjennison) [inline graphql/database/cause/causes/blackEquity/causeData.js:16]: Edit: Black Equity

File: graphql/database/cause/causes/blackEquity/causeData.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  isAvailableToSelect: false,
+  backgroundImageCategory: 'black-photographers',
+  charityId: 'b7a78116-9a62-4208-b016-03aae6a3414a', // bail project charity id
+  individualImpactEnabled: false,
+  impactVisits: null,
+  landingPagePath: '/blackEquity/',
```
Comment (kmjennison) [inline graphql/database/cause/causes/blackEquity/causeData.js:22]: Edit: /black-equity/

File: graphql/database/cause/causes/blackEquity/causeData.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  sharing: {
+    facebookButtonTitle:
+      'I joined Tab for Black Equity, and now every tab I open helps raise money for Thurgood Marshall College Fund and the Bail Project. Check it out - it’s free!',
+    imgCategory: 'blackEquity',
+    shareImage: 'blackEquity/blackEquityEmailInvite.svg',
+    sentImage: 'blackEquity/blackEquityEmailInvite.svg',
+    redditButtonTitle: 'Promote Black equity with each tab',
+    email: {
+      image:
+        'https://prod-tab2017-media.gladly.io/img/cause/backEquity/blackEquityEmailInvite.svg',
```
Comment (kmjennison) [inline graphql/database/cause/causes/blackEquity/causeData.js:52]: This image does not exist.

File: graphql/database/cause/causes/blackEquity/about.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+[The Bail Project](https://bailproject.org/) is a 501(c)(3) organization that works to combat mass incarceration through the money bail system. [The reality is](https://www.instagram.com/p/CLcbMG1gX0A/) that Black people are 25% more likely to be kept in jail pretrial without bail than other groups of people, and Black men have bail amounts 35% higher than white men. Through providing bail and additional pretrial support, The Bail Project helps restore the presumption of innocence to those being charged, reunites families, and challenges a system that criminalizes poverty and race.
+
+[The Thurgood Marshall College Fund](https://www.tmcf.org/) is a 501(c)(3) organization that advocates for students and public Historically Black Colleges and Universities (HBCUs), provides scholarships, and supports educational and career-driven programs. Its goal is to prepare the next generation of workforce talent through leadership development.
+
+For more information, please email us at contact @tabforacause.org.
```
Comment (kmjennison) [inline graphql/database/cause/causes/blackEquity/about.md:9]: Remove space before @

File: graphql/database/cause/causes/blackEquity/onboarding.step3.subtitle.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Ads on the new tab page raise money that we give to nonprofits. Most ads aren't good--but these ones are :)
```
Comment (kmjennison) [inline graphql/database/cause/causes/blackEquity/onboarding.step3.subtitle.md:1]: Change to em dash: Most ads aren't good—but these ones are :)

File: graphql/database/cause/causes/blackEquity/social.email.about.md [hunk 6]
```diff
@@ -0,0 +1 @@
+<div>Tab for #TeamSeas turns your web browser into a force for good. With each tab opened, ad revenue earned will support<a href="http://oceanconservancy.org"><span  style="color:#2690b3;text-decoration:underline;"> Ocean Conservancy</span></a> and <a href="http://theoceancleanup.com"><span style="color:#2690b3;text-decoration:underline;">The Ocean Cleanup</span></a>, working towards #TeamSeas’ goal of removing 30 million pounds of trash from oceans, rivers, and lakes around the world.</div><br>Learn more at <a href="http://tabforteamseas.org"><span style="color:#2690b3;text-decoration:underline;">tabforteamseas.org</span></a></div>
```
Comment (kmjennison) [inline graphql/database/cause/causes/blackEquity/social.email.about.md:1]: Fix copy.

File: graphql/database/cause/causes/blackEquity/causeData.js [hunk 7]
```diff
@@ -0,0 +1,89 @@
+// Cause: teamseas
```
Comment (kmjennison) [inline graphql/database/cause/causes/blackEquity/causeData.js:1]: Remove/fix comment

File: graphql/database/cause/causes/cats/social.email.about.md [hunk 8]
```diff
@@ -1,2 +1 @@
-<div>Tab for Cats turns your web browser into a paradise with daily cat background photos. On top of that, you’ll be helping shelter cats get adopted every time you open a new tab through ad revenue. 
-<br>Learn more at <a href="http://tabforcats.org"><span style="color:#2690b3;text-decoration:underline;">tabforcats.org</span></a></div>
+<div>Tab for Black Equity turns your web browser into a force for good. With each tab opened, ad revenue supports The Bail Project and Thurgood Marshall College Fund, uplifting the Black community through education, advocacy, and life-changing access to resources.</div><br>Learn more at <a href="http://tabforblackequity.org"><span style="color:#2690b3;text-decoration:underline;">tabforblackequity.org</span></a></div>
```
Comment (kmjennison) [inline graphql/database/cause/causes/cats/social.email.about.md:1]: tabforblackequity.org doesn't exist right now, use our pending landing page (HTTPS).

**Comment by amaliwanag2** [comment]
> Additional UI change: we need to refactor user menu spacing on new tab page so the absence of the impact counter isn't awkward. Maybe use 6px of spacing on either side of every item. Screen Shot 2022-01-12 at 11 13 32 AM

agreed, will follow that up

**Review by kmjennison — APPROVED** [review]
LGTM! 🦊

**Pull request merged**